src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -181,19 +189,19 @@ const SearchResult = ({ form, formMetaInfo, showPDFInfoBox }) => {
           </a>
         </dd>
       ) : null}
-      {showPDFInfoBox ? (
-        <dd className="vads-u-margin-bottom--2">
-          <va-alert status="info">
-            <div className="usa-alert-text vads-u-font-size--base">
-              <h3 slot="heading" className="vads-u-margin-top--0">
-                You'll need to download this form and open it in Adobe Acrobat
-                Reader
-              </h3>
-              <a href="https://www.va.gov/resources/what-if-im-having-trouble-opening-a-pdf/">
-                Get instructions for opening the form in Acrobat Reader
-              </a>
-            </div>
-          </va-alert>
+      {showPDFInfoVersionOne ? (
+        <dd className="find-forms-alert-message vads-u-margin-bottom--2 vads-u-background-color--primary-alt-lightest vads-u-display--flex vads-u-padding-y--4 vads-u-padding-right--7 vads-u-padding-left--3 vads-u-width--full">
+          <i aria-hidden="true" role="img" />
+          <span className="sr-only">Alert: </span>
```

Comment:
From the screenshot, the `<i />` above is an info icon? So should this say "Alert"?

Reply:
Yes!